app/serializers/api/public/v1/serializable_recruitment_cycle.rb

```diff
@@ -0,0 +1,13 @@
+module API
+  module Public
+    module V1
+      class SerializableRecruitmentCycle < JSONAPI::Serializable::Resource
+        type "recruitment_cycles"
+
+        attributes :application_start_date,
+                   :application_end_date,
+                   :year
```

Comment:
https://api2.qa.publish-teacher-training-courses.service.gov.uk/api-reference.html#schema-recruitmentcycleattributes

So is `year` a `string` or `integer`

Reply:
String it turns out 
![image](https://user-images.githubusercontent.com/25597009/90510193-6e297500-e152-11ea-9e72-69377db1ef35.png)